2008-11-19 0:09:49|[Error]无法从传输连接中读取数据: 远程主机强迫关闭了一个现有的连接。。
   在 System.Net.Sockets.NetworkStream.BeginRead(Byte[] buffer, Int32 offset, Int32 size, AsyncCallback callback, Object state)
   在 SagaLib.NetIO..ctor(Socket sock, Dictionary`2 commandTable, Client client, Boolean isgateway)
2008-11-19 0:09:49|[Error]无法访问已释放的对象。
对象名:“System.Net.Sockets.Socket”。
   在 System.Net.Sockets.Socket.get_RemoteEndPoint()
   在 SagaLib.NetIO..ctor(Socket sock, Dictionary`2 commandTable, Client client, Boolean isgateway)
   在 SagaGateway.GatewayClient..ctor(Socket mSock, Dictionary`2 mCommandTable)
   在 SagaGateway.GatewayClientManager.NetworkLoop(Int32 maxNewConnections)
   在 SagaGateway.Gateway.Main(String[] args)
2008-11-19 1:16:11|[Error]Failed... Trying again in 5sec
2008-11-19 1:16:11|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 2:08:11|[Error]Failed... Trying again in 5sec
2008-11-19 2:08:11|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 4:25:57|Warning:Invalid Client Version CRC!Kicking.... 
         Key:ED90AA25AE906FB36308C8523A4737A7E7B1FC6F
2008-11-19 5:54:28|[Error]Failed... Trying again in 5sec
2008-11-19 5:54:28|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 5:54:34|[Error]Failed... Trying again in 5sec
2008-11-19 5:54:34|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 5:54:49|[Error]Failed... Trying again in 5sec
2008-11-19 5:54:49|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 14:58:37|Warning:Invalid Client Version CRC!Kicking.... 
         Key:ED90AA25AE906FB36308C8523A4737A7E7B1FC6F
2008-11-19 16:12:28|[Error]无法从传输连接中读取数据: 远程主机强迫关闭了一个现有的连接。。
   在 System.Net.Sockets.NetworkStream.BeginRead(Byte[] buffer, Int32 offset, Int32 size, AsyncCallback callback, Object state)
   在 SagaLib.NetIO..ctor(Socket sock, Dictionary`2 commandTable, Client client, Boolean isgateway)
2008-11-19 16:12:28|[Error]无法访问已释放的对象。
对象名:“System.Net.Sockets.Socket”。
   在 System.Net.Sockets.Socket.get_RemoteEndPoint()
   在 SagaLib.NetIO..ctor(Socket sock, Dictionary`2 commandTable, Client client, Boolean isgateway)
   在 SagaGateway.GatewayClient..ctor(Socket mSock, Dictionary`2 mCommandTable)
   在 SagaGateway.GatewayClientManager.NetworkLoop(Int32 maxNewConnections)
   在 SagaGateway.Gateway.Main(String[] args)
2008-11-19 16:20:30|[Error]Failed... Trying again in 5sec
2008-11-19 16:20:31|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 16:20:35|[Error]Failed... Trying again in 5sec
2008-11-19 16:20:35|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 16:20:41|[Error]Failed... Trying again in 5sec
2008-11-19 16:20:41|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 18:03:39|[Error]Failed... Trying again in 5sec
2008-11-19 18:03:39|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 18:03:45|[Error]Failed... Trying again in 5sec
2008-11-19 18:03:45|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 21:12:31|[Error]Failed... Trying again in 5sec
2008-11-19 21:12:31|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 21:12:34|[Error]Failed... Trying again in 5sec
2008-11-19 21:12:34|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 21:54:02|[Error]无法从传输连接中读取数据: 远程主机强迫关闭了一个现有的连接。。
   在 System.Net.Sockets.NetworkStream.BeginRead(Byte[] buffer, Int32 offset, Int32 size, AsyncCallback callback, Object state)
   在 SagaLib.NetIO..ctor(Socket sock, Dictionary`2 commandTable, Client client, Boolean isgateway)
2008-11-19 21:54:02|[Error]无法访问已释放的对象。
对象名:“System.Net.Sockets.Socket”。
   在 System.Net.Sockets.Socket.get_RemoteEndPoint()
   在 SagaLib.NetIO..ctor(Socket sock, Dictionary`2 commandTable, Client client, Boolean isgateway)
   在 SagaGateway.GatewayClient..ctor(Socket mSock, Dictionary`2 mCommandTable)
   在 SagaGateway.GatewayClientManager.NetworkLoop(Int32 maxNewConnections)
   在 SagaGateway.Gateway.Main(String[] args)
2008-11-19 22:29:40|[Error]Failed... Trying again in 5sec
2008-11-19 22:29:41|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 22:29:46|[Error]Failed... Trying again in 5sec
2008-11-19 22:29:46|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
2008-11-19 22:29:52|[Error]Failed... Trying again in 5sec
2008-11-19 22:29:52|[Error]System.Net.Sockets.SocketException: 由于目标机器积极拒绝，无法连接。 117.88.11.80:6000
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   在 SagaGateway.LoginSession.Connect()
